packages/react/src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -70,19 +77,24 @@ const CheckboxGroup: React.FC<CheckboxGroupProps> = ({
     [`${prefix}--checkbox-group--invalid`]: !readOnly && invalid,
     [`${prefix}--checkbox-group--warning`]: showWarning,
     [`${prefix}--checkbox-group--slug`]: slug,
+    [`${prefix}--checkbox-group--decorator`]: decorator,
   });
 
-  // Slug is always size `mini`
-  let normalizedSlug;
+  // AILabel always size `mini`
+  let normalizedDecorator = slug ?? decorator;
```

Comment:
```suggestion
  let normalizedDecorator = React.isValidElement(slug ?? decorator)
      ? slug ?? decorator
      : null;
```
just rewrote this in my pr's too otherwise storybook throws a minify error when setting the `decorator` prop